.github/workflows/dlp-pipelines.yml

```diff
@@ -254,6 +254,81 @@ jobs:
           done
           echo "# records in input CSV file are: $rc_orig."
           echo "Verified number of rows in ${{env.INPUT_FILE_NAME}}_pub_sub: $row_count."
+  
+  de-identification-streaming-write:
+    needs:
+      - generate-uuid
+      - create-dataset
+
+    runs-on:
+      - self-hosted
+
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Run DLP Pipeline for streaming write
+        env:
+          DATASET_ID: ${{ needs.generate-uuid.outputs.output5 }}
+        run: |
+          gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV2 -Pargs=" \
+          --region=us-central1 --project=${{env.PROJECT_ID}} \
+          --tempLocation=gs://${{env.GCS_BUCKET}}/temp \
+          --numWorkers=1 --maxNumWorkers=2 --runner=DataflowRunner \
+          --filePattern=gs://${{env.GCS_BUCKET}}/streaming_write.csv --dataset=${{env.DATASET_ID}} \
+          --workerMachineType=n1-highmem-4 \
+          --inspectTemplateName=${{env.INSPECT_TEMPLATE_PATH}} \
+          --deidentifyTemplateName=${{env.DEID_TEMPLATE_PATH}} \
+          --batchSize=200000 --DLPMethod=DEID --serviceAccount=${SERVICE_ACCOUNT_EMAIL} \
+          --useStorageWriteApi --storageWriteApiTriggeringFrequencySec=2 --numStorageWriteApiStreams=2"
+
+      - name: Verify BQ table for streaming write
+        env:
+          DATASET_ID: ${{ needs.generate-uuid.outputs.output5 }}
+        run: |
+          not_verified=true
+          table_count=0
+          while $not_verified; do
+            table_count=$(($(bq query --use_legacy_sql=false --format csv 'SELECT * FROM `${{env.PROJECT_ID}}.${{env.DATASET_ID}}`.__TABLES__ WHERE table_id LIKE "streaming_write%"' | wc -l ) -1))
```

Comment:
Can we mention the file/table name 'streaming_write' as a variable at the top instead of hard-coding it?